github/testground.yml

```diff
@@ -32,15 +32,24 @@ repositories:
         - maintainers
     visibility: public
   github-mgmt:
+    # WARN: push+ access here should be treated exactly as cautiosly as org admin role
     branch_protection:
       master: {}
     files:
       CODEOWNERS:
-        content: >
-          *       @galargh
+        content: |
+          # The ipdx team is responsible for GitHub Management maintenance
+          * @testground/ipdx
+          
+          # The github-mgmt stewards team is responsible for triaging/reviewing configuration change requests
+          # The ipdx team is added here temporarily to witness use patterns in github-mgmt
+          /github/ipfs.yml @testground/github-mgmt-stewards @testground/ipdx
```

Comment:
```suggestion
          /github/testground.yml @testground/github-mgmt-stewards @testground/ipdx
```

Reply:
In practice do we need @testground/ipdx currrently given those members are are also @testground/github-mgmt-stewards ?

Reply:
One of the ways to reach us that we advertise is through tagging ipdx team in any issue. Since that only works within org bounds, I wanted to create ipdx teams in all the orgs for the sake of consistency.